packages/core/src/providers/apple.ts

```diff
@@ -158,17 +170,38 @@ export default function Apple<P extends AppleProfile>(
     name: "Apple",
     type: "oidc",
     issuer: "https://appleid.apple.com",
+    wellKnown: "https://appleid.apple.com/.well-known/openid-configuration",
+    checks: ["pkce"],
+    token: {
+      url: `https://appleid.apple.com/auth/token`,
+    },
```

Comment:
```suggestion
    wellKnown: "https://appleid.apple.com/.well-known/openid-configuration",
    checks: ["pkce"],
    token: {
      url: `https://appleid.apple.com/auth/token`,
    },
```

This is not necessary, as we use the `issuer` to discover the well-known endpoint for this metatdata.